spec/stores/jsonapidatastore.spec.js

```diff
@@ -295,8 +334,9 @@ describe('JsonApiDatastore', function() {
 
     it('should be able to create root level objects', function() {
       var person = {id: 1, name: 'John', bike: null, pets: []};
-      return expect(store.create('person', {id: 'some-uuid', name: 'John'}))
-        .to.become(person);
+      store.create('person', {id: 'some-uuid', name: 'John'}).then(function(result) {
```

Comment:
I don't think this test can ever fail since you're calling `expect` inside the promise. Also `result` won't become anything because it's not a promise.